native/Avalonia.Native/src/OSX/menu.mm

```diff
@@ -205,6 +205,17 @@ - (void)didSelectItem:(nullable id)sender
     }
 }
 
+HRESULT AvnAppMenuItem::SetIsVisible (bool isVisible)
+{
+    START_COM_CALL;
+    
+    @autoreleasepool
+    {
+        [_native setHidden:(isVisible ? NSOnState : NSOffState)];
```

Comment:
Shouldn't it be reversed?

Reply:
https://developer.apple.com/documentation/appkit/nsmenuitem/1514846-hidden?language=objc is a boolean, so I don't see why do we need NSOnState / NSOffState. 

Was it tested at all? Avalonia can run on macOS VMs for more than a year already.

Reply:
I didn't test on macos at all, since setting up macOS VMs on linux is hard.